one-container-alternative/Dockerfile

```diff
@@ -0,0 +1,104 @@
+ARG PY_VERSION=3.10
+
+# Base image used for all stages
+FROM python:${PY_VERSION} as base
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y \
+    && apt-get install -yq \
+      inotify-tools \
+      iputils-ping \
+      lsb-release \
+      net-tools \
+      postgresql \
+      sendmail \
+      snmp \
+      tree \
+      vim \
+    && apt-get clean \
+    && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
+
+
+# stage for python dependencies
+FROM base as python_dependencies
+
+ARG PYTHONUNBUFFERED=1
+ARG PIP_NO_CACHE_DIR=off
+ARG PIP_NO_WARN_SCRIPT_LOCATION=0
+ARG PIP_DISABLE_ROOT_WARNING=ignore
+ARG PIP_DISABLE_PIP_VERSION_CHECK=1
+
+COPY python_dependencies/requirements.txt /tmp/requirements.txt
+
+RUN pip install --upgrade -r /tmp/requirements.txt 
+
+
+# stage for yangsuite setup
+FROM python_dependencies as app_setup
+
+EXPOSE 443 3000 8080 8086 8443 8480 9339 50051 50052 57344 57345
```

Comment:
Only need 8480 for YS GUI and then 57500-57501 for gRPC Telemetry receiver

Reply:
Fixed